src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -297,6 +293,28 @@ private void RecalculateTimeOffsetLocked()
             this.logger.LogTrace("(-):{0}={1}", nameof(this.timeOffset), this.timeOffset.TotalSeconds);
         }
 
+        /// <summary>
+        /// Gets the median offset of all the inbound samples (limited to 1 less than the OutboundToInboundWeightRatio per outbound) and the outbound samples with a weighting of the OutboundToInboundWeightRatio.
+        /// </summary>
+        /// <returns>Median offset in seconds.</returns>
+        private double GetMedianOffsetInSecondsFromInboundAndWeightedOutbound()
+        {
+            var inboundOffsets = this.inboundTimestampOffsets.Select(s => s.TimeOffset.TotalSeconds);
```

Comment:
unvar on 302, 303, 310

Reply:
please explain why?

Reply:
it is code style policy rule. The code is supposed to follow the policies. If you disagree with the policy, you can create an issue suggesting an improvement in the policy. But unless the policy is changed, everyone is supposed to follow the policy

Reply:
In this project we try to use Var only when it is evident that final type is without looking somewhere else, this saves time of the reader of the code

Reply:
If is it is just "because it is the policy" then fine, I will do it, but I think it is a good idea to challenge these things.

Reply:
It is because of the policy here, so that we don't need to explain reasons for that policy in every PR. But that policy has very good reasons behind its that, so at The end it is not just because of the policy